packages/entity-server/src/routes/hierarchies/types.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Flattable, Flattened } from '../../types';
+
+export type HierarchyFields = Readonly<{
+  id: string;
```

Comment:
I skipped all entity fields that seem to be empty for project entities.

Also left out:
* `parent_id`
* `type`
* `bounds` - to keep things simple. Also, not sure if we want to include geographical data in the hierarchy resource anyway

Reply:
Might've been fine leaving `type`, but all good calls 👍